src/Tribe/Editor/Configuration.php

```diff
@@ -24,8 +28,17 @@ public function hook() {
 	 * @return array
 	 */
 	public function editor_config( $editor_config ) {
-		$tec = empty( $editor_config['events'] ) ? array() : $editor_config['events'];
+		$tec                     = empty( $editor_config['events'] ) ? [] : $editor_config['events'];
 		$editor_config['events'] = array_merge( (array) $tec, $this->localize() );
+
+		$post_objects                  = empty( $editor_config['post_objects'] ) ? [] : $editor_config['post_objects'];
+		$editor_config['post_objects'] = array_merge(
+			(array) $post_objects,
+			[
+				TEC::POSTTYPE => ( new Event_Object() )->data()
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 391f1ffd1ff5379bb21640920288ec22</sub>